src/Data/Fin/Base.agda

```diff
@@ -240,6 +240,14 @@ punchIn zero    j       = suc j
 punchIn (suc i) zero    = zero
 punchIn (suc i) (suc j) = suc (punchIn i j)
 
+-- The surjection f(i,j) such that f(i,i) = f(i,i+1) = i
+
+pinch : ∀ {m} → Fin m → Fin (suc m) → Fin m
```

Comment:
We usually use `n` for natural numbers when we're only using one of them. I realise we're not that consistent in this particular file!